kokkosConfig.h

```diff
@@ -5,12 +5,15 @@
 
 #ifdef DIGI_KOKKOS_SERIAL
 using KokkosExecSpace = Kokkos::Serial;
+using KokkosMemSpace = Kokkos::HostSpace;
```

Comment:
You can actually pass the `KokkosExecSpace` as the memory space template argument, so I'd expect these type aliases to be unnecessary (at least to leading order).

Reply:
sure, will do.

Reply:
fixed